packages/grid/stories/examples/Default.tsx

```diff
@@ -0,0 +1,259 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story } from '@storybook/react';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+interface IDefaultStoryProps {
+  columns: number;
+  rows: number;
+  offset: number;
+  size: number;
+  breakpoint: string;
+  debug: boolean;
+  wrap: 'wrap' | 'nowrap' | 'wrap-reverse' | undefined;
+  textAlign: 'start' | 'end' | 'center' | 'justify' | undefined;
+  gutters: false | 'xxs' | 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl';
+  justifyContent: 'start' | 'end' | 'center' | 'between' | 'around' | undefined;
+  alignItems: 'start' | 'end' | 'center' | 'baseline' | 'stretch' | undefined;
+}
+
+export const Default: Story<IDefaultStoryProps> = ({
+  columns,
+  gutters,
+  debug,
+  rows,
+  justifyContent,
+  alignItems,
+  wrap,
+  textAlign,
+  offset,
+  breakpoint,
+  size
+}) => {
+  const computedSize = size >= columns ? columns : size;
+  const computedOffset =
+    size > 1 ? Math.min(offset, columns - size) : Math.min(offset, columns - 1);
+
+  return (
+    <Grid>
+      <Row>
+        <Col size="8">
```

Comment:
Everything looks good to me except for this surrounding `Grid` which should be removed so we can observe how the grid behaves across various screen sizes.